src/psyclone/tests/test_files/dynamo0p3/1.6.3_single_invoke_multiple_derived_types.f90

```diff
@@ -11,17 +11,17 @@ program single_invoke
   ! Description: four kernels specified in an invoke call where the
   ! one integer argument is obtained from different components of two
   ! different objects
-  use testkern_one_int_scalar, only: testkern_type
+  use testkern_one_int_scalar_mod, only: testkern_one_int_scalar_type
```

Comment:
Please update the copyright notice in this file.

Reply:
Done